Loop/Managers/LoopDataManager.swift

```diff
@@ -1698,7 +1700,23 @@ extension LoopDataManager {
                     return self.delegate?.roundBolusVolume(units: units) ?? units
                 }
 
-                let maxAutomaticBolus = min(iobHeadroom, maxBolus! * LoopConstants.bolusPartialApplicationFactor)
+                // Create dosing strategy based on user setting
+                let dosingStrategy: DosingStrategy = UserDefaults.standard.bool(forKey: "applyLinearRampToBolusApplicationFactor")
```

Comment:
Storing this in user defaults is fine for algorithm experiments, but please put "experimental" in the key name. If this ever makes it as a standard options of the Loop algorithm, it will go into LoopSettings